docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -43,66 +42,16 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
- 
-[Learn more about how linking works](github-transition-guide.md#technical-information).  
-
-### Costs
- 
-Ministries won’t be charged for upgraded GitHub licenses.  However, ordering IDIRs for contractors may cost and need expense approvals. We have more information about IDIRS on our [IDIR transition guide](github-transition-guide.md#idirs). 
-  
-### Directions to link your accounts before July 30
-
-The video shows the process, starting from the banner link on the bcgov GitHub organization, to the end:
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/-cfhUY_or8s?si=q0pjaxYd4FBhNZ0s" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
-
-
-#### For employees 
-
-1. Have any two-factor devices, and personal GitHub login and IDIR login ready for linking 
-2. Log into GitHub and follow the prompts on the pop up to connect your GitHub and IDIR accounts together. 
-3. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled.
-
-    - [Learn more about how to do this](github-transition-guide.md#resetting-github-keys). 
- 
-That’s it, your accounts are now linked.  
- 
-#### For contractors 
- 
-Contractors without IDIRs can find more information on our [IDIR transition guide](github-transition-guide.md#idirs). It can take several days to complete depending on your ministry. We recommend doing with this step as soon as possible to avoid access disruptions.  
- 
-After getting an IDIR, follow the steps below: 
-
-1. Have any two-factor device, and personal GitHub login and IDIR login ready for linking 
-2. Log into GitHub and follow the prompts on the pop up to connect your GitHub and IDIR accounts together. 
-3. If linking fails, your IDIR configuration is not compatible with our GitHub SSO. You must [request an upgrade to a P2 level IDIR account](github-transition-guide.md#idirs).  
-4. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled.
-    - [Learn more about how to do this](github-transition-guide.md#resetting-github-keys). 
- 
-That’s it, your accounts are now linked.  
  
-### Directions to link your account after July 30 
+### Directions to link your account
 
-Employees and contractors locked out after **July 30th, 2024** can follow the steps below to regain access. If you are a contractor without an IDIR, follow the steps in the [IDIR transition guide](github-transition-guide.md#idirs) before proceeding:
+If you are a contractor without an IDIR, follow the steps in the [IDIR transition guide](github-transition-guide.md#idirs) before proceeding:
  
 1. If you have a valid IDIR, ask anyone with access to the organization to re-invite you.
```

Comment:
This should be self serve now and shouldn't require someone else inviting them in.  Maybe something like: 
```suggestion
1. If you have a valid IDIR, follow the instructions at (https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github/#bcgov) to join the bcgov organization.
```